src/features/ITGovernance/Admin/GRBReview/GRBReviewForm/GRBReviewFormStepWrapper/index.tsx

```diff
@@ -45,7 +46,8 @@ function GRBReviewFormStepWrapper<
   children,
   grbReview,
   onSubmit,
-  requiredFields = true
+  requiredFields = true,
+  disabled = false
```

Comment:
I think this prop can be removed now that the validation was fixed.

Reply:
I think I was using that prop to render the help text atop the form for the required fields

Reply:
https://github.com/CMS-Enterprise/easi-app/blob/c2d9ab7cd60a2548388d231228b82d1269a9aef4/src/features/ITGovernance/Admin/GRBReview/GRBReviewForm/GRBReviewFormStepWrapper/index.tsx#L257-L260

Reply:
I should have been more specific - I was just talking about the `disabled` prop that was added in this PR. The existing `requiredFields` prop is used in other steps so definitely need to keep that!